lib/utils.js

```diff
@@ -102,12 +107,20 @@ function pick2list(options, config) {
 
 // Custom Processing - Combining all processes
 function processOption(options, config, hasWhitespace) {
-  const newValues = merge(options.map(option => split(option, ',', hasWhitespace)))
-  const oldValues = (config ? (typeof config === 'string' ? split(config, ',', false) : config) : [])
-  const diff = newValues.filter(value => oldValues.indexOf(value) < 0).filter(value => value)
+  const newValues = merge(options.map((option) => {
+    return split(option, ',', hasWhitespace)
+  }))
+  const oldValues = (config
+    ? (typeof config === 'string'
+      ? split(config, ',', false)
+      : config)
+    : [])
+  const diff = newValues.filter((value) => {
+    return oldValues.indexOf(value) < 0
+  }).filter((value) => { return value })
```

Comment:
I think this is the same thing as `.filter(Boolean)` 

what are we trying to filter out here? empty strings?

Reply:
I believe the intention was to remove empty strings.